packages/zstd.rb

```diff
@@ -0,0 +1,17 @@
+require 'package'
+
+class Zstd < Package
+  description 'Zstandard - Fast real-time compression algorithm'
+  homepage 'http://www.zstd.net'
+  version '1.3.4'
+  source_url 'https://github.com/facebook/zstd/archive/master.tar.gz'
```

Comment:
We should use the versioned source at https://github.com/facebook/zstd/archive/v1.3.4.tar.gz.